src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -4,14 +4,17 @@
 
   <PropertyGroup>
     <Description>Various helpers for writing tests that use ASP.NET Core.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore</PackageTags>
     <EnableApiCheck>false</EnableApiCheck>
   </PropertyGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
```

Comment:
Can we add this to common.props? Perhaps as `$(TargetFrameworkMoniker)'=='NETFramework'?